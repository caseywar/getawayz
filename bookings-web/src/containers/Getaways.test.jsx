import React from 'react';
import { render, screen } from '@testing-library/react';
import Getaways from './Getaways';
import { MemoryRouter } from 'react-router';

describe('Getaways container', () => {
    it('renders a list of places on the web page', async () => {
        render(
            <MemoryRouter>
                <Getaways />
            </MemoryRouter>
        )

        const ul = await screen.findAllByRole( 'list', {name: 'places'});
        expect(ul).toMatchSnapshot();
    })
})