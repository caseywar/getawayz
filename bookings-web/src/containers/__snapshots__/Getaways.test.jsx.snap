// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Getaways container renders a list of places on the web page 1`] = `
Array [
  <ul
    aria-label="places"
    class="list"
  />,
]
`;
